doc/source/contributing_code.txt

```diff
@@ -131,11 +131,14 @@ all issues have been resolved.
 <li>Programmers begin work on a feature or fix by branching from
 develop.
 
-<li>When branch is ready, it is submitted to code review.
+<li>When a branch is ready, it is submitted to code review.
 
 <li>When code review is complete, and the changes are approved, the PR
 is merged into the develop branch.
 
+<li>Mutliple merges into the develop branch may take place between
+test cycles. (???)
+
```

Comment:
Since we are going to wait for the cdash tests (which are the tests on yellowstone and other actual HPC platforms), then we can only integrate to master once per cdash testing cycle. (Every 6 hours, I believe.)

Since I routinely submit 2-4 PRs a day, the math does not work. Either we integrate multiple PRs routinely, or else I have fewer PRs (and therefor less isolation of changes in different PRs).

So I think the best way is to do multiple PRs at a time. If something fails I will just have to roll back all of them and figure it out.